index.src.html

```diff
@@ -130,8 +130,16 @@ <h3 id="examples">Example</h3>
   </pre>
 
   Before blocking this insecure request as mixed content, the user agent
-  performs an HSTS priming request to the root of the resource's host, as
-  follows:
+  consults its <a lt="matches the HSTS cache">HSTS cache</a> for the origin of
+  the resource.
+
+  If the origin of the resource matched the HSTS cache, the user agent can then
```

Comment:
Nit: The cache is also negative; that is, it stores HSTS priming failures as well as successes. "Matching" just means that we don't need to do a priming request, not that we can safely upgrade the request.